File: PS/Programmers/시저암호.py
def solution(s, n):
    s=list(s)
    for i in range(len(s)):
        if s[i].isupper():
            s[i] = chr((ord(s[i])+n-65)%26+65)
        if s[i].islower():
            s[i] = chr((ord(s[i])+n-97)%26+97)
    return ''.join(s)

File: PS/Programmers/시저 암호.cc
#include <string>
#include <vector>

using namespace std;
string solution(string s, int n) {
	string answer = "";
	for (int i = 0; i < s.size(); i++)
	{
		if (s[i] == 32)
		{
			continue;
		}
		if (s[i] < 91)
		{
			s[i] = (((s[i] - 65) + n) % 26) + 65;
		}
		else if (s[i] > 96)
		{
			s[i] = (((s[i] - 97) + n) % 26) + 97;
		}
	}
	return s;
}